pages/moderation.vue

```diff
@@ -1,22 +1,57 @@
 <template>
-    <div data-testid="moderation-page" class="flex flex-row h-screen">
-        <!-- This button exists solely for testing that the states work and how you can implement it on other components. -->
-        <!-- <button @click="store.setActiveScreen(ModerationScreen.editSubmission)">Change state!</button> -->
-        <ModLeftNavbar />
-        <div class="w-full flex flex-col items-stretch">
-            <ModTopbar />
-            <ModMainContent />
+    <Suspense>
+        <div data-testid="moderation-page" class="flex flex-row h-screen">
+            <!-- This button exists solely for testing that the states work and how you can implement it on other components. -->
+            <!-- <button @click="store.setActiveScreen(ModerationScreen.editSubmission)">Change state!</button> -->
+            <ModLeftNavbar />
+            <div class="w-full flex flex-col items-stretch">
+                <ModTopbar />
+                <ModMainContent />
+            </div>
         </div>
-    </div>
+        <!-- loading state via #fallback slot -->
+        <template #fallback>
+            Checking auth...
```

Comment:
I know this is still WIP, but let's add this to the localization files too 🙏🏾

Reply:
will do, I'm not even sure this fallback is even showing yet 😆